src/caches/abstractbeliefpropagationcache.jl

```diff
@@ -283,3 +283,69 @@ function update(
 )
   return update(Algorithm(alg), bpc; kwargs...)
 end
+
+function rescale_message(bp_cache::AbstractBeliefPropagationCache, partitionpair)
+  return rescale_messages(bp_cache, typeof(partitionpair)[partitionpair])
+end
+
+function rescale_messages(bp_cache::AbstractBeliefPropagationCache)
+  return rescale_messages(bp_cache, partitionpairs(bp_cache))
```

Comment:
Should we rename `partitionpairs` to `partitionedges`? That name is throwing me off, I was confused by what it meant.

Reply:
Yeah good point, I will rename it